app/views/shared/_navigation.html.erb

```diff
@@ -1,14 +1,24 @@
-<div class="nav">
-  <ul class="nav nav-tabs">
-    <a class="navbar-brand" href="/">Waddams Emporium</a>
-    <li role="presentation">
-      <% if logged_in? %>
-      <%= link_to "Logout", logout_path, method: :delete,  class: "nav-item nav-link"%>
-      <% else %>
-      <%= link_to "Login", login_path, class: "nav-item nav-link"%>
-      <%= link_to "Create Account", new_user_path, class: "nav-item nav-link"%>
-      <% end %>
-    </li>
-  </ul>
+<div class="container">
+  <div class="nav">
+    <ul class="nav nav-tabs">
+      <a class="navbar-brand" href="/">Waddams Emporium</a>
+      <li role="presentation">
+        <% if logged_in? %>
+          <%= link_to "Welcome, #{current_user.username}...", dashboard_path, class: "nav-item nav-link" %>
+          <%= link_to "Checkout", orders_path, class: "nav-item nav-link" %>
```

Comment:
I think we want to only show checkout if a user is logged in and has items in the cart.